js/dashboard.js

```diff
@@ -61,12 +61,12 @@ fetch('https://api.weatherbit.io/v2.0/forecast/daily?lat=-45.874&lon=170.503&key
 .then(response =>         
     response["data"].filter(day => day.datetime == formatDate(testSaturday(new Date())) || day.datetime == formatDate(testSunday(new Date())) || day.datetime == getToday())
     .forEach((data, i) => {
-
     //date that needs to be formatted to say the day
-
+        console.log('data', data)
```

Comment:
Is this needed?

Reply:
All done man @Jordann-Hand  Got them all